OC-6 Button class takes a string to display a label over the button

hschallh opened the pull request:
closes #13

hschallh committed:
Button class takes a string to display a label over the button on display()

GideonBuckwalter commented on Button.pde:
```diff
@@ -2,17 +2,21 @@ class Button{
   float posX, posY;
-  
-  Button(float _posX, float _posY){
+  int textSize = 16;
```
I think a better name for this variable would be `fontSize` because it controls the font directly, and because `textSize` is a builtin function name.

GideonBuckwalter replied:
Everything else looks great!

hschallh replied:
Agreed on the renaming.

joezcool02 commented:
still waiting on the variable rename then this is good to go @hschallh

hschallh committed:
Updated textSize variable for the button class

As per @GideonBuckwalter suggestion, renamed textSize to fontSize, to not conflict with the reserved function textSize().

hschallh commented:
fin.

joezcool02 commented:
don't forget to commit with the issue number!
